packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -166,6 +168,27 @@ class GestureRecognizerFactoryWithHandlers<T extends GestureRecognizer> extends
 /// ```
 /// {@end-tool}
 ///
+/// {@tool snippet}
+///
+/// This example of a [Container] wraps a [GestureDetector] widget.
+/// Since the [GestureDetector] does not have a child it takes on the size of
+/// its parent making the entire area of the surrounding [Container] clickable.
+/// When tapped the [Container] turns yellow by setting the `_color` field:
+///
+/// ```dart
+/// Container(
```

Comment:
Can we specify a size for the Container to make this example a little clearer? Or maybe use a SizedBox?

Reply:
Done